tinygrad/tensor.py

```diff
@@ -42,6 +42,7 @@ class Tensor:
 
   def __init__(self, data:Union[int, float, list, tuple, LazyBuffer, np.ndarray], device:Optional[str]=None, dtype:Optional[DType]=None, requires_grad:Optional[bool]=None):
     assert dtype is None or isinstance(dtype, DType), f"invalid dtype {dtype}"
+    if not isinstance(data, (int, float)) and data.__class__ not in (list, tuple, LazyBuffer, np.ndarray): raise RuntimeError(f"can't create Tensor from {data}")
```

Comment:
use an assert, not if raise runetimerror + use one big isinstance.

readability >>> all